packages/docs/cookbook/testing.md

```diff
@@ -102,6 +102,14 @@ expect(store.someAction).toHaveBeenCalledTimes(1)
 expect(store.someAction).toHaveBeenLastCalledWith()
 ```
 
+If you are not using Jest (for example, you are using vitest), you will need to provide a [createSpy](https://pinia.vuejs.org/api/interfaces/pinia_testing.testingoptions.html#createspy) to `createTestingPinia()`, like:
```

Comment:
It actually works if you have `globals: true` with vitest. Let's:

- Add a new subsection about actions being mocked
- Say they are automatically mocked in jest and in vite with globals: true
- Show how to add your own createSpy function

Reply:
Hmm, I have this is my vite.config.ts:

```
    test: {
      globals: true,
      environment: 'happy-dom',
    } as VitestUserConfig,
```

let me investigate.

Reply:
@posva I think a new version of @pinai/testing needs to created and published. The behavior with `vi` was changed/fixed in [this commit](https://github.com/vuejs/pinia/commit/44f0762f93bf6ea389dd47574cbf4172989f616b) 11 days ago, but it's not published.

Reply:
Well maybe not. I see that [vi is just a reference to vitest](https://github.com/vitest-dev/vitest/blob/c1fa19dfc2ccf6c27b2289dc829ec29c53ec66b6/packages/vitest/src/integrations/utils.ts#L86), so either of those symbols should work. I'll have to find another reason why this isn't working on my machine.

(Incidentally, I can see the `vi` global in my test file, but still get the error when calling `createTestingPinia`.

Reply:
ok @posva I'm sorry for all the noise here, but yes, the reason that you still need to supply `createSpy` with the released version of @pinia/testing (0.0.9) is that support for detecting `vi`/`vitest` was [only added 11 days ago,](e4638ef34c16487b2f35b0de09800f6dcfd5a980) and there has not been a release of @pinia/testing since then.

I'm unfamiliar with your release schedule so perhaps that is intentional. 

I'll prepare this PR to update the documentation in anticipation of a new release being available (it looks like there are comments in the code along the lines of what you'd like the docs to say so I'll use those as a guide).

Reply:
Sure! I probably haven’t released because there are a few things to change before